tests/tt_eager/python_api_testing/unit_testing/misc/test_all_gather.py

```diff
@@ -191,7 +192,11 @@ def test_all_gather_on_t3000_post_commit_looping(
     num_iters,
     use_program_cache,
     function_level_defaults,
+    enable_async,
 ):
+    for device in all_devices:
```

Comment:
I didn't want to clog the pipelines since the all-gather tests already take a while. Its redundant testing, for this feature if we test on all configs. I get your point though, I'll just move it to the run function.

Reply:
I see - does the async have a separate isolated pipeline? In that case should we just have a couple tests run there? (Otherwise yea, I'd say enable globally)